std_msgs/Int16 x
std_msgs/Int16 y
std_msgs/Int16 z
